python/cudf/cudf/_lib/pylibcudf/lists.pyx

```diff
@@ -94,3 +100,99 @@ cpdef Column concatenate_list_elements(Column input, bool dropna):
         ))
 
     return Column.from_libcudf(move(c_result))
+
+
+cpdef Column contains(Column input, ColumnOrScalar search_key):
+    """Create a column of bool values indicating whether
+    the search_key is contained in the input.
+
```

Comment:
Can we link back to the cpp docs here and below? [example](https://github.com/rapidsai/cudf/blob/branch-24.08/python/cudf/cudf/_lib/pylibcudf/binaryop.pyx#L33)